casanovo/config.py

```diff
@@ -96,10 +96,22 @@ def __init__(self, config_file: Optional[str] = None):
                 # Remap deprecated config entries.
                 for old, new in _config_deprecated.items():
                     if old in self._user_config:
-                        self._user_config[new] = self._user_config.pop(old)
+                        warning_msg = (
+                            f"Depreciated config option '{old}' "
+                            "is no longer in use"
+                        )
+
+                        if new is not None:
+                            self._user_config[new] = self._user_config.pop(old)
+                            warning_msg = (
+                                f"Deprecated config option '{old}' "
+                                f"remapped to '{new}'"
+                            )
+                        else:
```

Comment:
**todo:** Can you add a test for this situation [here](https://github.com/Noble-Lab/casanovo/blob/save-best/tests/unit_tests/test_config.py#L42)?